prod-deploy/kv-mall-manifest/kv-mall.yaml

```
@@ -170,10 +169,13 @@ spec:
         app: inventory
     spec:
       containers:
-        - name: inventory
-          image: registry.odigos.io/kv-mall-inventory:v0.3
-          ports:
-            - containerPort: 8080
+      - name: inventory
+        image: dev/inventory:dev
```

Note the dev tag on production deployment:
```suggestion
        image: registry.odigos.io/kv-mall-inventory:v0.xx
```